beacon/blockchain/pruning.go

```diff
@@ -38,7 +38,7 @@ func (s *Service[
 	}
 
 	// prune deposit store
-	start, end = depositPruneRangeFn(
+	start, end = depositPrunRangeFn(
```

Comment:
is this from a different branch? intentional?

Reply:
It carried over from Polar's pub/sub cleanup branch, indeed.